src/shop/accounts/verify.js

```diff
@@ -101,7 +101,7 @@ bloombox.shop.verify = function(email,
 
   rpc.send(function(response) {
     if (done) return;
-    if (response != null) {
+    if (response !== null) {
```

Comment:
lots of small bugs like this fixed, where before, a `!=` comparison could have resulted in `response` being implicitly cast to `null`, if it was, say, `-1` or another value of similar, _falsy_ nature.

however, we want to match against `null` and only `null`, without JavaScript doing any silent "magic" (read: behaving like an asshole). so it's an exact match with `!==`.